_layouts/author.html

```diff
@@ -9,89 +9,62 @@ <h2>{{ page.position }}</h2>
 
 <h2>Skillset</h2>
 
-<h3>Languages</h3>
+{%- assign langs_framework_misc = "langs,frameworks,misc" | split: "," -%}
+{%- assign split_size = 5 -%}
+{%- assign colwidth = 100 | divided_by: split_size -%}
 
-<table>
-  <tr> 
-    {% for lang in page.langs -%}
-    <th colspan="2">
-      {{ site.data.skillset["langs"][lang.name].name }}
-      {%- if lang.ver -%}
-      ({{lang.ver | join: ","}})
-      {%- endif -%}
-    </th>
-    {%- endfor %}
-  </tr>
-  <tr>
-    {% for lang in page.langs -%}
-    {% if lang.self and lang.want %} {% assign cs = 1 %} {% else %} {% assign cs = 2 %} {% endif %}
-    {% if lang.self %}<th colspan="{{cs}}">Self</th>{% endif %}
-    {% if lang.want %}<th colspan="{{cs}}">Want</th>{% endif %}
-    {%- endfor %}
-  </tr>
-  <tr>
-    {% for lang in page.langs -%}
-    {% if lang.self and lang.want %} {% assign cs = 1 %} {% else %} {% assign cs = 2 %} {% endif %}
-    {% if lang.self %}<td colspan="{{cs}}">{{lang.self}}</td>{% endif %}
-    {% if lang.want %}<td colspan="{{cs}}">{{lang.want}}</td>{% endif %}
-    {%- endfor %}
-  </tr>
-</table>
+{%- comment -%} Get the skills from the default language page {%- endcomment -%}
+{% for stype in langs_framework_misc -%}
+    <h3> {{ site.data.skillset["titles"][stype] }} </h3>
 
-<h3>Frameworks</h3>
+    {%- assign default_language = site.defaults[0].values.lang -%}
+    {%- comment -%} Addressing by short_name avoid issues in case a person has different names in different languages (yes, this is a thing) {%- endcomment -%}
+    {%- assign base_author = site.authors | where: "lang", default_language | where: "short_name", page.short_name -%}
 
-<table>
-  <tr>
-    {% for fw in page.frameworks -%}
-    <th colspan="2"> {{ site.data.skillset["frameworks"][fw.name].name }} {%- if fw.ver -%} ({{fw.ver | join: ","}})
-      {%- endif -%} </th>
-    {%- endfor %}
-  </tr>
-  <tr>
-    {% for fw in page.frameworks -%}
-    {% if fw.self and fw.want %} {% assign cs = 1 %} {% else %} {% assign cs = 2 %} {% endif %}
-    {% if fw.self %}<th colspan="{{cs}}">Self</th>{% endif %}
-    {% if fw.want %}<th colspan="{{cs}}">Want</th>{% endif %}
-    {%- endfor %}
-  </tr>
-  <tr>
-    {% for fw in page.frameworks -%}
-    {% if fw.self and fw.want %} {% assign cs = 1 %} {% else %} {% assign cs = 2 %} {% endif %}
-    {% if fw.self %}<td colspan="{{cs}}">{{fw.self}}</td>{% endif %}
-    {% if fw.want %}<td colspan="{{cs}}">{{fw.want}}</td>{% endif %}
-    {%- endfor %}
-  </tr>
-</table>
+    {%- comment -%} ceil(a/b) = (a+(b-1))/b with integer division, avoiding floating point {%- endcomment -%}
+    {% assign splits = base_author[0][stype].size | plus: split_size | minus: 1 | divided_by: split_size %}
 
-<h3>Miscellaneous</h3>
+    {% for i in (1..splits) -%}
+        {%- assign start_idx = forloop.index0 | times: split_size -%}
+        {%- assign length = base_author[0][stype].size | minus: start_idx -%}
```

Comment:
that is exactly what I was trying to do, but I gave up thinking that making CSS handle this would have been easier. do you think is worth checking that route too?